modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/CodeTemplates.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.project.templates;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParameterException;
+
+import com.liferay.project.templates.internal.CodeGenerator;
+import com.liferay.project.templates.internal.util.FileUtil;
+
+import java.io.File;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * @author Gregory Amerson
+ */
+public class CodeTemplates {
+
+	public static void main(String[] args) throws Exception {
+		CodeTemplatesArgs codeTemplatesArgs = new CodeTemplatesArgs();
+
+		JCommander jCommander = new JCommander(codeTemplatesArgs);
+
+		try {
+			Path jarPath = FileUtil.getJarPath();
+
+			if (Files.isRegularFile(jarPath)) {
+				jCommander.setProgramName("java -jar " + jarPath.getFileName());
+			}
+			else {
+				jCommander.setProgramName(CodeTemplates.class.getName());
+			}
+
+			jCommander.parse(args);
+
+			new CodeTemplates(codeTemplatesArgs);
+		}
+		catch (ParameterException pe) {
+			System.err.println(pe.getMessage());
+		}
+	}
+
+	public CodeTemplates(CodeTemplatesArgs codeTemplatesArgs) throws Exception {
+		File destinationDir = codeTemplatesArgs.getDestinationDir();
```

Comment:
The `destinationDir` should be moved to `L61` to follow the `as used` principle.